stacks-coordinator/src/coordinator.rs

```diff
@@ -57,6 +57,8 @@ pub enum Error {
     /// "Bitcoin Sighash Error"
     #[error("Bitcoin Sighash Error")]
     BitcoinSighash(#[from] SighashError),
+    #[error("Command Receiver Error: {0}")]
+    CommandReceiverError(#[from] std::sync::mpsc::RecvError),
```

Comment:
The receiver should only error if the sender has been dropped ([link](https://doc.rust-lang.org/std/sync/mpsc/struct.Receiver.html#method.recv)). I think it's fine to panic in that case because it should not be possible and is completely out of the callers control. I am okay with the explicit error too. However, I suggest we're clear about the implication:

```suggestion
    #[error("Command sender disconnected unexpectedly: {0}")]
    UnexpectedSenderDisconnect(#[from] std::sync::mpsc::RecvError),
```